src/screens/explore-screen/components/ColorTile.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { ReactNode } from 'react'
+
+import {
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  ViewStyle
+} from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import IconAudioRewardsPill from 'app/assets/images/iconAudioRewardsPill.svg'
+import Text from 'app/components/text/Text'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { font } from 'app/styles'
+import { ThemeColors } from 'app/utils/theme'
+
+type ColorTileProps = {
+  style?: ViewStyle
+  title: string
+  link: string
+  description?: string
+  gradientColors?: string[]
+  gradientAngle?: number
+  shadowColor?: string
+  shadowOpacity?: number
+  icon?: React.FC
+  emoji?: ReactNode
+  goToRoute: (route: string) => void
+  isIncentivized?: boolean
+}
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    shadowContainer: {
+      shadowOffset: { height: 2, width: 0 },
+      shadowRadius: 4
+    },
+
+    gradientContainer: {
+      borderRadius: 8
+    },
+
+    colorTile: {
+      backgroundColor: 'rgba(255,255,255,0.1)',
+      borderRadius: 8,
+      height: 200,
+      overflow: 'hidden',
+      padding: 16,
+      position: 'relative'
+    },
+    hasEmoji: {
+      height: 128,
+      textAlign: 'center'
+    },
+
+    title: {
+      ...font('heavy'),
```

Comment:
wondering how yall feel about putting this utility into the theme itself, so its like `theme.typography.h2` or something?

Reply:
I think something like this could be good, would be interested to see how it is implemented / how it is used

Reply:
mui's theme was the inspiration for me, they have a ton of utils looks like: https://mui.com/customization/default-theme/

Reply:
Oooh thats some good stuff 👌